### DISCUSSION TITLE
Integration of multiple single-cell multiomics samples for GRN construction

### DISCUSSION THREAD

**Discussion by Ajeet1699** [aertslab/scenicplus#194] [Announcements]
Hi,

Thanks for building the wonderful tool.
I have multiple single cell multiomics (same cells) samples (6 diseased and 6 control individuals) and I want to construct single GRN for each cell type. I am following `"https://scenicplus.readthedocs.io/en/latest/pbmc_multiome_tutorial.html" `which I understand was run only on one sample. What is the best way to integrate the multiple samples (`cell ranger aggr` or by `Seurat`) for building GRN? I can build an integrated .h5 object for scRNA seq data but for ATAC, there are separate "atac_fragments.tsv" files for each samples, how to handle these?

And I am also wondering that will I have to build the separate GRNs for diseased and control samples? or is these a way to compare these in Scenic+?

Thanks in advance

**Comment by SeppeDeWinter** [comment]
Hi @Ajeet1699 

Thank you for your kind words.
I moved your issue to the discussions tab because it is more a question for advice(and not a problem with the code).

Indeed, for the scRNA-seq part build a single h5 file containing all the samples.

For ATAC, you can also generate a single cisTopic object for all the samples. See: https://pycistopic.readthedocs.io/en/latest/Cortex_pycisTopic.html for more information.

For the GRN inference part. I would run it on all samples at the same time (not separately on disease and control samples). The reason for this is that the inference part needs contrasts to be able to find relevant networks. After running it once on all samples you can start comparing diseased and control states.

I hope this helps.

Best,

Seppe

**Reply by YayanFeng55** [reply]
Dear Seppe,

Thanks for your nice suggestions on how to construct GRNs between disease and control samples. One quick question: how can we compared diseased and control states after we finish running Scenic+? Should we add cell states information (disease or control) in cell metadata table (celldata_mel.tsv)?

Thanks a lot in advance
Yayan

**Reply by SeppeDeWinter** [reply]
Hi @YayanFeng55 

I'm afraid there is no one answer on how to do the comparison... We don't have functions for that in the SCENIC+ package. 

All the best,

Seppe
